### PR TITLE
LPS-200741 Replace AUI() in dynamic-data-mapping-form-web

### PR DESCRIPTION
**Pull request by DiegoHu97**
[Link to ticket](https://liferay.atlassian.net/browse/LPS-200741)

## Steps 

- Go to Content & Data > Forms > Create a Form and save it
- Go to Site Builder > Pages > Create a Content Page 
- Add a Form widget > Go to it's configuration 
- Select the form created

https://github.com/liferay-echo/liferay-portal/assets/91208451/7c9d9a88-0aee-4501-85fb-da1c7aac9b7d

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/configuration.jsp [hunk 1]
```diff
@@ -148,30 +148,34 @@ DDMFormInstance selFormInstance = DDMFormInstanceServiceUtil.fetchFormInstance(f
...
+			document.getElementById(
+				'<portlet:namespace />formInstanceId'
+			).value = formInstanceId;
 
-			document.<portlet:namespace />fm.<portlet:namespace />formInstanceId.value = formInstanceId;
-
-			var formInstanceHolder = A.one('.displaying-form-instance-id-holder');
+			const formInstanceHolder = document.getElementsByClassName(
+				'displaying-form-instance-id-holder'
+			)[0];
```
Comment (p2kmgcl) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/configuration.jsp:157]: :disguised_face: please use a querySelector, just to remove the [0]

**Review by p2kmgcl — CHANGES_REQUESTED** [review]

**Comment by DiegoHu97** [comment]
@p2kmgcl Updated! Thanks 😄

**Comment by p2kmgcl** [comment]
ci:forward

**Comment by p2kmgcl** [comment]
ci:stop

**Comment by p2kmgcl** [comment]
ci:forward

**Pull request closed without merging**